ekingan/debug-drills

```diff
@@ -0,0 +1 @@
+Subproject commit 8021bafdf4baf3e4555a4c9acb316b6f6ceecf47
```

Comment:
Please don't copy other repos into your homework directory or this happens: I can't see any of the files inside this folder. :-(  
Since it's not homework this time I don't need it; so don't worry about this one.  ;-) 

If you do need to copy another repo into homework you can do the following:
1) copy the repo in 
2) **immediately** go into that repo and delete it's hidden .git directory (make sure its not the `23-modules/.git`)
3) add the repo files